v1/client.go

```diff
@@ -60,6 +60,12 @@ type Client struct {
 
 // DefaultClient returns a client that connects to the Atlas API.
 func DefaultClient() *Client {
+	atlasEndpoint := os.Getenv("ATLAS_ADDRESS")
```

Comment:
Only chose `ATLAS_ADDRESS` because that's what Terraform uses. I didn't know about the env var in the others.